### ISSUE TITLE
The `fullScreen` is not functioning properly.

### ISSUE THREAD

**Issue by ibaoyun** [leanflutter/window_manager#458]
During `waitUntilReadyToShow`, the `fullScreen` is set to `true`. However, when the window is restored using `setFullScreen(false)`, it only displays the background color without any content and the taskbar icon also disappears. Additionally, calling `windowManager.minimize();` has no effect either. The platform in use is Windows.

**Comment by Arvinth-Krishna** [comment]
Yes, I'm facing the same issue, when I switch from one display to another.

**Comment by humam-alBasha** [comment]
You can skip the issue until it is resolved . 

`
  const WindowOptions windowOptions = WindowOptions(
    size: Size(800, 600),
    center: true,
    backgroundColor: Colors.transparent,
    skipTaskbar: false,
    // titleBarStyle: TitleBarStyle.hidden,
  );
  windowManager.waitUntilReadyToShow(windowOptions, () async {
    await windowManager.show();
    await windowManager.focus();
    await windowManager.setFullScreen(true);
  });
`

**Comment by humam-alBasha** [comment]
In the setFullScreen function , setSize is Future function need await
    // (Windows) Force refresh the app so it 's back to the correct size
    // (see GitHub issue #311)
    if (Platform.isWindows) {
      final size = await getSize();
      setSize(size + const Offset(1, 1));
      setSize(size);
    }

**Comment by Markus43** [comment]
Just to add: There's a working plugin for windows here: https://pub.dev/packages/fullscreen_window - might be interesting to compare the code and evaluate the differences.

**Comment by temeipu-offical** [comment]
> 补充一点：这里有一个适用于 Windows 的插件：https: [//pub.dev/packages/fullscreen_window](https://pub.dev/packages/fullscreen_window) - 对比一下代码并评估一下差异可能会很有意思。

https://github.com/jakky1/fullscreen_window/issues/8

**Comment by lijy91** [comment]
window_manager 0.5.x is no longer maintained: from 0.6.0 window_manager is rebuilt on [nativeapi](https://github.com/libnativeapi/nativeapi) (Flutter package: [nativeapi](https://pub.dev/packages/nativeapi)), and its platform code moves there.

This hasn't been checked against nativeapi yet, so it's now on the verification list in libnativeapi/nativeapi#77. If you can reproduce it with nativeapi's `Window` API (or window_manager 0.6.0 once released), please add your reproduction there.
